packages/valory/skills/abstract_round_abci/test_tools/base.py

```diff
@@ -143,9 +143,6 @@ def fast_forward_to_behaviour(
     ) -> None:
         """Fast forward the FSM to a behaviour."""
         next_behaviour = {s.behaviour_id: s for s in behaviour.behaviours}[behaviour_id]
-        assert (  # nosec
-            next_behaviour is not None
-        ), f"Behaviour {behaviour_id} not found"
```

Comment:
`behaviour.behaviours: Set[Type[BaseBehaviour]]`, that is to say one gets a `KeyError`, `None` is impossible.